File: CA.ERP.WebApp/Dto/User/UserView.cs
using CA.ERP.Domain.UserAgg;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CA.ERP.WebApp.Dto.User
{
    public class UserView : DtoViewBase
    {
        public string Username { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public UserRole Role { get; set; }
        public List<UserBranchView> UserBranches { get; set; }
    }
}

File: CA.ERP.WebApp/ClientApp/src/app/home/management/user/model/user.view.ts
export interface UserView {
    id : number;
    status : number;
    createdAt : string;
    createdBy : string;
    updatedAt : string;
    updatedBy : string;
    username : string;
    firstName : string;
    lastName : string;
    role : number;
    userBranches : []
}